website/blog/ctas.yml

```diff
@@ -45,3 +45,9 @@
   subheader: Watch our annual dbt Launch Showcase to hear from our executives and product leaders about the latest features landing in dbt. 
   button_text: Watch the talks
   url: https://www.getdbt.com/resources/webinars/2025-dbt-cloud-launch-showcase
+- name: vs_code_extension
+  header: VS Code extension
+  subheader: The free dbt VS Code extension is the best way to develop locally in dbt.
+  button_text: Install free extension
+  icon: vsce
+  url: /docs/install-dbt-extension
```

Comment:
We're working on a new landing page for all things Fusion, which will have tiles for the platform, local install, and the extension - would it make sense, when that's available, to put that link here so we can drop them in the middle of all things Fusion? Would that be too many clicks?

@runleonarun would like to hear your thoughts as well

Reply:
@matthewshaver This can stay the same because I'm creating a snippet to share content in both places.